docs/reference/scientific-workflows.md

```diff
@@ -0,0 +1,43 @@
+
+A scientific workflow is a series of interconnected tasks or computational steps 
+that are designed to solve a specific problem or address a scientific question. 
+
+[Nextflow](https://www.nextflow.io/) and 
+[Cromwell](https://cromwell.readthedocs.io/en/stable/) are examples of tools that allow 
+researchers to define, automate, and execute such workflows.
+
+Scientific workflow can be beneficial when an analysis involves multiple steps, is 
+complex, needs to be repeated, involves different platforms or environments, or requires
+collaboration.
+
+## Workflows in Geneweaver
+Geneweaver uses scientific workflows to define analysis tools. This allows researchers
+to define complex analysis tools that can be used by other researchers in the Geneweaver
+web application.
+
+For single step analysis tools, the workflow definition can be inferred by the 
+Geneweaver web application. 
+
+### Why Use Workflows?
+Geneweaver uses scientific workflows as part of its analysis model for many of the 
+same reasons scientists use them in their research. Utilizing workflow also allows
+Geneweaver to utilize the same tools that scientists use to define and execute their
+workflows.
+
+In general, scientific workflows can be beneficial for the following reasons:
+
+1) Reproduce scientific results: Workflows can be shared with others and executed on 
```

Comment:
1 & 4 have similar titles, different conclusions. Maybe change this section to "Share methods"? Or combine 1&4 into a single section on reproducibility.